temporalcli/commandsgen/commands.yml

```diff
@@ -1647,9 +1647,8 @@ commands:
         default: 7233
       - name: http-port
         type: int
-        description: | 
-          Port for the HTTP API service.
-          Default is off.
+        description: Port for the HTTP API service.
+        default: 7243
```

Comment:
Can we get confirmation that enabling HTTP by default is only for an internal Nexus completion callback mechanism? If so, can we instead use an in-memory approach instead of binding this port for everyone by default?

Reply:
It's used for callbacks and for all other Nexus RPCs in the [spec](https://github.com/nexus-rpc/api/blob/main/SPEC.md).

Available to be called from external clients or internally by the history service Nexus machinery.

Reply:
[3aaf3a2](https://github.com/temporalio/cli/pull/734/commits/3aaf3a2623ea7fae5cd3010e8d6e80ed9a065475)